packages/packages.csv

```diff
@@ -119,6 +119,7 @@ Microsoft.Data.Sqlite,ship,include
 Microsoft.Data.Sqlite.Core,ship,include
 Microsoft.DotNet.Watcher.Tools,ship,exclude
 Microsoft.EntityFrameworkCore,ship,include
+Microsoft.EntityFrameworkCore.ApplicationInsights,ship,include
```

Comment:
Is this added here: https://github.com/aspnet/MetaPackages/blob/dev/build/common.props#L75?